src/pages/GalleryDetails/components/GalleryRenderer.tsx

```diff
@@ -0,0 +1,27 @@
+// React
+import { useState } from 'react';
+
+// Types
+import { Gallery } from 'types';
+
+// Components
+import ImageDialog from './ImageDialog';
+import ImageGrid from './ImageGrid';
+
+type GalleryRendererProps = {
```

Comment:
```suggestion
export type GalleryRendererProps = {
```

Eksporter props når komponenten også eksporteres